Article/Article.js

```diff
@@ -1,20 +1,30 @@
 // Because classes are not hoised you will need to start your code at the bottom of the page.  Look for the comment "START HERE"
 
 class Article {
-  constructor(domElement) {
+  constructor(article) {
     // assign this.domElement to the passed in domElement
-    this.domElement;
+    this.article = article;
     // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.expandButton = article.querySelector('.expandButton');
     // Using your expandButton reference, update the text on your expandButton to say "expand"
-    
+    if (this.article.classList.contains('article-open')){
+      this.expandButton.textContent = 'Click to close!'
+    } else{
+      this.expandButton.textContent = 'Click to open!';
+    }
     // Set a click handler on the expandButton reference, calling the expandArticle method.
-    
+    this.expandButton.addEventListener('click', () => this.expandArticle());
   }
 
   expandArticle() {
     // Using our reference to the domElement, toggle a class to expand or hide the article.
-
+    this.article.classList.toggle('article-open');
+    if (this.article.classList.contains('article-open')){
+      this.expandButton.textContent = 'Click to close!'
+    } else{
+      this.expandButton.textContent = 'Click to open!';
+    }
+    TweenMax.to(article, 2, {left: 600});
```

Comment:
it will animate your article to the left. I don't think you want that. What direction is your element expanding?

Read the docs- https://greensock.com/docs/TweenMax/static.fromTo()